common.css

```diff
@@ -23,8 +23,8 @@ body {
     display: none;
     z-index: 1;
     position: absolute;
-    top: 32px;
-    left: -32px;
+    top: 3.2rem;
+    left: -3.2rem;
     background-color: var(--white);
     border-radius: 10px;
```

Comment:
you can try using utility classes for border-radius, as there are same repetitive styles of border-radius throughout all your commits(just a suggestion), otherwise ret of your code looks good to me , no other changes from my side, you can go ahead with the merging process!!